ballerina/tests/test.bal

```diff
@@ -193,6 +196,55 @@ function testDeleteObject() {
     }
 }
 
+@test:Config {
+    dependsOn: [testListObjects]
+}
+function testCreateMultipartUpload() returns error? {
+    log:printInfo("amazonS3Client->createMultipartUpload()");
```

Comment:
Do we need this log lines ? I think test function name should convey the meaning of the test rather than adding additional logs.

Reply:
I added it because it was there in other tests. Although, it does help to see the test execution order.